app/controllers/IndexController.scala

```diff
@@ -50,14 +50,23 @@ class IndexController @Inject()(
           case Some(userAnswers) if userAnswers.get(AcknowledgementReferencePage).isDefined =>
             Future.successful(Redirect(routes.RegistrationSentController.onPageLoad()))
           case _ =>
+            val cacheData = charitiesKeyStoreService.getCacheData(request)
```

Comment:
is it possible to move this as part of  for comprehension, please remove extra space/line

Reply:
sure - was doing something with it, but forgot to remove it after.